src/Microsoft.Azure.SignalR.Management/RestApiProviderExtensions.cs

```diff
@@ -28,5 +30,20 @@ public static async Task OrTimeout(this Task task, CancellationToken cancellatio
                 }
             }
         }
+
```

Comment:
Why rename the file? `OrTimeout` is used in `NegotiateProcessor` and `ServiceHubContextBuilder`. It seems nothing to do with `WithTracingIdAsync`

Reply:
by accident...